### PR TITLE
Add virtual integration: Whisper

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/93183
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/4382
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by c0ffeeca7 — COMMENTED** [review]
Does _protocol_ need to be capitalized here?

**Comment by frenck** [comment]
> Does _protocol_ need to be capitalized here?

It is an integration name, which depends on naming from upstream. Some upstream projects use all caps or any other combination, often part of branding. I never diverge from the naming given from those. It is inherited from:

https://github.com/home-assistant/core/blob/b993fe1c9d6aafc9fbf6b1cb8785098e8a22cc56/homeassistant/components/wyoming/manifest.json#L3

If that needs to be changed or is incorrect, it would require changes to Core first (and not really in scope of this PR, as this PR is about adding Whisper, not changing or adjusting the Wyoming Protocol integration).

../Frenck

**Review by c0ffeeca7 — APPROVED** [review]

File: source/_integrations/whisper.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Whisper
+description: Connect and control your Whisper instance using the Wyoming Protocol integration
+ha_category:
+  - Voice
+ha_domain: whisper
+ha_release: '2023.5'
```
Comment (MartinHjelmare) [inline source/_integrations/whisper.markdown:7]: Is it this integration release or wyoming release?
```suggestion
ha_release: '2023.6'
```

**Review by MartinHjelmare — APPROVED** [review]

File: source/_integrations/whisper.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/whisper.markdown:7]: wyoming, it follow the original integration

**Pull request merged**